inference/core/interfaces/http/http_api.py

```diff
@@ -780,6 +780,10 @@ async def legacy_infer_from_request(
                     description="Shared secret used to authenticate requests to the inference server from internal services (e.g. to allow disabling inference usage tracking via the `countinference` query parameter)",
                     include_in_schema=False,
                 ),
+                disable_preproc_auto_orient: Optional[bool] = False,
+                disable_preproc_contrast: Optional[bool] = False,
+                disable_preproc_grayscale: Optional[bool] = False,
+                disable_preproc_static_crop: Optional[bool] = False,
```

Comment:
are these passed on the query string? do we need to have `Query(...` in that case?

Reply:
Good catch! This works as is, but using the `Query(...)` function allows us to add docs. Just pushed a fix~